keyboards/keychron/k6/rgb/v2/config.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2022 Philip Mourdjis <philip.j.m@gmail.com>
+ * Copyright 2023 KeijoMika (https://github.com/KeijoMika)
+ * Copyright 2023 Santanu Paik (https://github.com/ITNerd69)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+#define SN32_RGB_MATRIX_ROW_PINS {C6, C5, C4, C9, C8, C7, C12, C11, C10, B13, C14, C13, B14, B15, D3}
+#define DIP_SWITCH_PINS {D5, D4}
```

Comment:
this can be data driven now. See https://github.com/SonixQMK/qmk_firmware/commit/98530cad3ba8733d8a100e0bc5f3cf47339c4b3e